classes/Services/AddThisScriptManager.php

```diff
@@ -164,6 +165,28 @@ function setAsync($enabled) {
   }
 
   /**
+   * Get a array with all addthis_plugin_info values.
+   */
+  private function getJsAddThisPluginInfo() {
+    if ($this->addthis->isPluginInfoEnabled()) {
+      $plugin_info = array(
+        'info_status' => 'enabled',
+        'cms_name' => 'Drupal',
+        'cms_version' => VERSION,
+        'plugin_name' => 'AddThis',
+        'plugin_version' => '7.x-4.0-alpha7',
```

Comment:
You can get the current module version through the system_get_info() 

https://api.drupal.org/api/drupal/modules%21system%21system.module/function/system_get_info/7